tests/test_unit_file_metadata_init.cpp

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2018 Snowflake Computing, Inc. All rights reserved.
+ */
+
+#include <fstream>
+#include <FileMetadataInitializer.hpp>
+#include "FileMetadata.hpp"
+#include "FileCompressionType.hpp"
+#include "utils/test_setup.h"
+#include <unordered_set>
+#include <iostream>
+#include <map>
+
+#define FILES_IN_DIR "file1.csv", "file2.csv", "file3.csv", "file4.csv", "file1.gz"
+
+
+typedef ::Snowflake::Client::FileCompressionType FileCompressionType;
+
+void getDataDirectory(std::string& dataDir)
+{
+  const std::string current_file = __FILE__;
+  std::string testsDir = current_file.substr(0, current_file.find_last_of('/'));
+  dataDir = testsDir + "/data/";
```

Comment:
ok for now to run tests on Linux. But keep in mind that windows will needs double backslashes.

Reply:
Sure.